src/server/utils/getPubSub.ts

```diff
@@ -3,11 +3,11 @@ import {RedisPubSub} from 'graphql-redis-subscriptions'
 
 getDotenv()
 
-let pubsub
+let pubsub: RedisPubSub | undefined
 const getPubSub = () => {
   if (!pubsub) {
     pubsub = new RedisPubSub({
-      connection: process.env.REDIS_URL
+      connection: process.env.REDIS_URL as any
```

Comment:
nit: shouldn't this be `as string`?

Reply:
nah, the typings expect an object, but we're using the shorthand which is a string